src/projectBank2/Account.java

```diff
@@ -0,0 +1,77 @@
+package projectBank2;
+
+import java.math.BigDecimal;
+
+public class Account {
+
+    private BigDecimal myPLN;
+    private BigDecimal myGBP;
+    private BigDecimal myUSD;
+    private String kontoPLN;

```

Comment:
Same as with myXXX, add only one "konto" and rename to e.g accountNumber. Also switch to English in code, you will write code in English in Nokia :).